perfkitbenchmarker/providers/ibmcloud/ibmcloud_network.py

```diff
@@ -153,7 +156,7 @@ def _Create(self):
     else:
       raise errors.Error('IBM Cloud ERROR: Failed to create vpc')
 
-  def CreateSubnet(self, vpcid):
+  def CreateSubnet(self, vpcid: str):
```

Comment:
Nit: typing for the return, in this case " -> None:"

Reply:
Thanks for pointing that out, I will continue to make improvements going forward.